### ISSUE TITLE
Create Telegram Instant View templates

### ISSUE THREAD

**Issue by TimJentzsch** [GameFeeder/GameFeeder#69]
**Describe the solution you'd like**
We need a [Telegram Instant View](https://instantview.telegram.org/) template for every important provider page, especially the [Dota Patch Page](http://www.dota2.com/patches).

**Comment by zachkont** [comment]
The dota patch page turns out terrible in instant view because by default all images provided in the template are full-width "banner" images. I do not know of a way to make it look like an inline icon and I assume there isn't one. This is how it looks like without images: 
![isntantview](https://user-images.githubusercontent.com/17274058/60832483-a545fc80-a1c4-11e9-9c5b-97753182fb12.jpg)

I am not sure if it's worth adding that and we at least need to provide the original link along with the IV link

**Comment by TimJentzsch** [comment]
There is an **Icon** type that might do what we are looking for, found [here](https://instantview.telegram.org/docs).
Additionally, I would recommend making the hero names and maybe the spell names bold.

**Comment by zachkont** [comment]
I have created a template which covers most pages under the `dota2.com` domain. Specifically:

- blog.dota2.com
- dota2.com/updates
- dota2.com/news

to use it we just need to add the following link in the messages:

`https://t.me/iv?url={{url}}&rhash=a8aa3892b2f818`

and it should work. I have also submitted the template for review so that it is automatically used without the custom url but I have no idea if they even review them or when that will happen.
